Makefile

```
@@ -128,7 +128,13 @@ cert_manager: remove_cert_manager
 remove_cert_manager:
 	- $(CLUSTER_CLIENT) delete -f $(CERT_MANAGER_MANIFEST)
 
-
+.PHONY: check_cert_manager
+check_cert_manager:
+ifeq ($(shell kubectl api-versions | grep -c '^cert-manager.io/v1$$'), 0)
+ifneq ($(DISABLE_SERVICE_TLS), true)
+	$(error cert-manager is not installed, install using "make cert_manager" or disable TLS for services by setting DISABLE_SERVICE_TLS to true)
+endif
+endif
```

I tweaked this check to use the shell instead of being evaluated by Make when parsing the Makefile. What do you think?
```suggestion
       @api_versions=$$(kubectl api-versions) &&\
       if [ $$(echo "$${api_versions}" | grep -c '^cert-manager.io/v1$$') -eq 0 ]; then if [ "$${DISABLE_SERVICE_TLS}" != "true" ]; then\
                       echo 'cert-manager is not installed, install using "make cert_manager" or disable TLS for services by setting DISABLE_SERVICE_TLS to true' >&2\
                       && exit 1;\
               fi;\
       fi
```